src/cmd/linuxkit/push_gcp.go

```diff
@@ -26,6 +26,7 @@ func pushGcp(args []string) {
 	familyFlag := flags.String("family", "", "GCP Image Family. A group of images where the family name points to the most recent image. *Optional*")
 	nameFlag := flags.String("img-name", "", "Overrides the name used to identify the file in Google Storage and the VM image. Defaults to the base of 'path' with the '.img.tar.gz' suffix removed")
 	nestedVirt := flags.Bool("nested-virt", false, "Enabled nested virtualization for the image")
+	uefiCompatible := flags.Bool("uefi-compatible", false, "Enabled UEFI_COMPATIBLE feature for the image. It is required for enable of vTPM.")
```

Comment:
> "Enabled UEFI_COMPATIBLE feature for the image. It is required for enable of vTPM."

How about, `"Enable UEFI_COMPATIBLE feature for the image, required to enable vTPM."`